index.html

```diff
@@ -62,7 +62,7 @@ <h2 class="pldoc-heading pldoc-tab-heading pldoc-element-preview-title">{{post.t
                             <div class="pldoc-settings">
                                 <div class="pldoc-setting pldoc-setting-sizer">
                                     <label for="iconFontSlider" class="sr-only">Drag to resize icons</label>
-                                    <input id="iconFontSlider" type="range" min="12" max="144" step="12" value="24" role="slider" aria-valuemin="12" aria-valuemax="144" aria-valuenow="24">
+                                    <input id="iconFontSlider" type="range" min="12" max="144" step="12" value="48" role="slider" aria-valuemin="12" aria-valuemax="144" aria-valuenow="48">
```

Comment:
Yeah, that's cool. One NIT: can we make the initial size of these icons larger. Before sliding, they seem smaller than they once were.